src/app/typescript/v5/[...slug]/page.tsx

```diff
@@ -0,0 +1,29 @@
+import notFound from "../../../not-found";
+import { fetchTypeScriptDoc } from "../../../references/components/TDoc/fetchDocs/fetchTypeScriptDoc";
+import { RootTDoc } from "../../../references/components/TDoc/Root";
+import { getSlugToDocMap } from "../../../references/components/TDoc/utils/slugs";
+
+type PageProps = { params: { slug: string[] } };
+
+export default async function Page(props: PageProps) {
+	const doc = await fetchTypeScriptDoc("v5");
+	const slugToDoc = getSlugToDocMap(doc);
+	const docSlug = props.params.slug?.join("/");
+
+	if (!docSlug) {
```

Comment:
Instead of creating pages for all slugs - we should filter out and only include the slugs we can use in the sidebar.tsx file

Reply:
same q as above